src/logic/community-roles.ts

```diff
@@ -1,48 +1,35 @@
-export type CommunityPermission =
-  | 'edit_info'
-  | 'add_remove_places'
-  | 'accept_reject_requests'
-  | 'ban_players'
-  | 'send_invitations'
-  | 'edit_settings'
-  | 'delete_community'
-  | 'assign_roles'
+import { CommunityRole, CommunityPermission } from '../types/entities'
 
-export type CommunityRole = 'owner' | 'moderator' | 'member'
+export const OWNER_PERMISSIONS: CommunityPermission[] = [
+  'edit_info',
+  'add_remove_places',
```

Comment:
what do you think about splitting these grouped permissions in two?
`add_remove_places` and `accept_reject_requests` just to be more granular and prevent migrations in the future if a new role able to accept request but not to reject them appears in the wild